inventory/models.py

```diff
@@ -32,13 +32,26 @@ def __str__(self):
         return self.name
 
 
+# Example of status: Archived, Reserved, Lent, Borrowed, others
+class ItemStatus(models.Model):
```

Comment:
Maybe add a description field for status model in order to document better their intent and purpose